src/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To Do List</title>
+    <script src="https://kit.fontawesome.com/a3ad5bee92.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <div class="container">
+        <h1>Today's To Do</h1>
+        <div class="add-area">
+            <input type="text" id="add-task" placeholder="Add to your list...">
+            <i class="fas fa-plus fa-2x"></i>
+        </div>
+        <ul>
+
+        </ul>
+        <button type="button">Clear all completed</button>
+    </div>
+</body>
```

Comment:
Good work so far, However,  you may try using semantic tags to ensure good HTML practices

Reply:
Thank you for the review. I fixed it